zips/zip-0227.rst

```diff
@@ -307,33 +287,79 @@ For the ``IssueBundle``:
+We use the notation :math:`\mathsf{issued\_assets}(\mathsf{AssetBase}).\!\mathsf{balance}` and :math:`\mathsf{issued\_assets}(\mathsf{AssetBase}).\!\mathsf{final}` to access, respectively, the elements of the tuple stored in the global state for a given :math:`\mathsf{AssetBase}\!`.
+If :math:`\mathsf{issued\_assets}(\mathsf{AssetBase}) = \bot\!`, it is assumed that :math:`\mathsf{issued\_assets}(\mathsf{AssetBase}).\!\mathsf{balance} = 0` and :math:`\mathsf{issued\_assets}(\mathsf{AssetBase}).\!\mathsf{final} = 0\!`.
+
+For any Asset represented by :math:`\mathsf{AssetBase}\!`:
 
-- Validate the issuance authorization signature, :math:`\mathsf{issueAuthSig}\!`, on the SIGHASH transaction hash, :math:`\mathsf{SigHash}\!`, by invoking :math:`\mathsf{IssueAuthSig}.\!\mathsf{Validate}(\mathsf{ik}, \mathsf{SigHash}, \mathsf{issueAuthSig})\!`.
+- :math:`\mathsf{issued\_assets}(\mathsf{AssetBase}).\!\mathsf{balance} \in \{0 .. \mathsf{MAX\_ISSUE}\}` stores the amount of the Asset in circulation, computed as the amount of the Asset that has been issued less the amount of the Asset that has been burnt. 
+- :math:`\mathsf{issued\_assets}(\mathsf{AssetBase}).\!\mathsf{final} : \mathbb{B}` is a Boolean that stores the finalization status of the Asset (i.e.: whether the :math:`\mathsf{finalize}` flag has been set to :math:`1` in any preceding issuance transaction for the Asset). The value of :math:`\mathsf{issued\_assets}(\mathsf{AssetBase}).\!\mathsf{final}` for any :math:`\mathsf{AssetBase}` cannot be changed from :math:`1` to :math:`0\!`.
 
-For each ``IssueAction`` in ``IssueBundle``:
+The maximum total supply of any issued Custom Asset is denoted by the constant :math:`\mathsf{MAX\_ISSUE} := 2^{64} - 1 \!`. 
 
-- check that :math:`0 < \mathtt{assetDescSize} \leq 512\!`.
-- check that :math:`\mathsf{asset\_desc}` is a string of length :math:`\mathtt{assetDescSize}` bytes.
 
-- retrieve :math:`\mathsf{AssetBase}` from the first note in the sequence and check that :math:`\mathsf{AssetBase}` is derived from the issuance validating key :math:`\mathsf{ik}` and :math:`\mathsf{asset\_desc}` as described in the `Specification: Asset Identifier`_ section.
-- check that :math:`\mathsf{issued\_assets(AssetBase).final} \neq 1` in the global state.
-- check that every note in the ``IssueAction`` contains the same :math:`\mathsf{AssetBase}` and is properly constructed as :math:`\mathsf{note} = (\mathsf{g_d}, \mathsf{pk_d}, \mathsf{v}, \text{ρ}, \mathsf{rseed}, \mathsf{AssetBase})\!`.
+Management of the Global Issuance State
+---------------------------------------
 
-If all of the above checks pass, do the following:
+The issuance state, that is, the :math:`\mathsf{issued\_assets}` map, MUST be updated by a node during the processing of any transaction that contains burn information, or an issuance bundle.
+The issuance state is chained as follows:
 
-- For each note, 
+- The issuance state for the first block in the OrchardZSA protocol is the empty map.
```

Comment:
`in a protocol that supports OrcahrdZSA`

Reply:
How about:
`The issuance state for the first block post the activation of the OrchardZSA protocol is the empty map.`